be1-go/network/server.go

```diff
@@ -62,7 +62,7 @@ func NewServer(hub hub.Hub, addr string, port int, st socket.SocketType, log zer
 		log:     log,
 	}
 
-	path := fmt.Sprintf("/%s/%s", hub.Type(), st)
+	path := fmt.Sprintf("/sockets/%s", st)
```

Comment:
why not just the following? 😄 
```suggestion
	path := fmt.Sprintf("/%s", st)
```